<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialSetting</class>
 <widget class="QWidget" name="SerialSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>116</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="4">
    <widget class="QComboBox" name="cbFlowCtrl">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>无</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>HW</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SW</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="2" rowspan="4">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="cbAnalyse">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>84</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>无</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>数字</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>$T=数字 ;</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QComboBox" name="cbCheckBit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>无</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>奇</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>偶</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_27">
     <property name="text">
      <string>数据协议：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QComboBox" name="cbDataBit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>35</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_26">
     <property name="text">
      <string>编码格式：</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="label_25">
     <property name="text">
      <string>控制流：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_23">
     <property name="text">
      <string>校验位：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>数据位：</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="cbEncode">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>54</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>无</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>GBK</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Big5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>UTF8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>UTF16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ISO8859</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="cbBaudrate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>11</number>
     </property>
     <item>
      <property name="text">
       <string>300</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>900</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>28800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>56000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>128000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>230400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>460800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>921600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1382400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1500000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2000000</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="label_24">
     <property name="text">
      <string>停止位：</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QComboBox" name="cbStopBit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1.5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_29">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>波特率：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_28">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>串口号：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="lbSerial">
     <property name="text">
      <string>COMX</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
